service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseUrlService.java

```diff
@@ -6,27 +6,16 @@
 import org.springframework.stereotype.Service;
 
 import static org.apache.commons.lang3.StringUtils.isBlank;
-import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.CASE_TYPE;
-import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.JURISDICTION;
 
 @Service
 @RequiredArgsConstructor(onConstructor = @__({@Autowired}))
 public class CaseUrlService {
```

Comment:
i think this class is now not really needed, this can be moved to AbstractEmailContentProvider

Reply:
i tried, there are couple of issues i am facing. It can be moved to AbstractEmailContentProvider but i can't test an Abstract class (not exactly true, there is a way using Mockito). Another issue is, AbstractEmailContentProviderTest is extending in lot of classes, if i remove CaseUrlService, i have to remove @BeforeEach in abstract class, which means i have to modify all extended classes. 

I will explore but please let me know what you think!